options-admin.php

```diff
@@ -634,7 +634,7 @@ function AttributeAccessMethod()
 <?php       break;
 	case 'logging' :
 		$constant = false;
-		list( $shib_logging, $shib_logging_constant ) = shibboleth_getoption( 'shibboleth_logging', true, false, true );
+		list( $shib_logging, $shib_logging_constant ) = shibboleth_getoption( 'shibboleth_logging', array(), false, true );
```

Comment:
I'm thinking this should be `array(), true, true`. All the other ones that have a default value of array() are immediately followed by `true`

Reply:
You're correct, I'll fix this in the dot release as well.